CSC307Fall2018/team-too
# added Orders page

# Summary
* Orders can be viewed from /admin/orders
  * closes #43 
* Orders can be filtered by order status or delivery type
  * closes #46

# QA
* [x] verify that order ID still routes to an individual order page
* [x] verify that all orders in database appear on orders page
  * I would run `query{orders{id}}` from the graphql playground and verify that the same number of IDs and the same IDs appear
### Status Filtering
* [x] verify that filtering for a single order status filters properly
* [x] verify that filtering for multiple order status filters properly
* [x] verify that filtering for all order statuses shows all orders
* [x] verify that filtering for no order status shows all items *after disabling all order status filters*
### Order Type Filtering
* [x] verify that filtering for a single order type filters properly
* [x] verify that filtering for multiple order type filters properly
* [x] verify that filtering for all order types filters shows all orders
* [x] verify that filtering for no order type shows all items *after disabling all order type filters*
### Multiple Filters Filtering
* [x] verify that 1 status filter to many type filters actually filters properly
* [x] verify that many status filter to many type filters actually filters properly
* [x] verify that many status filter to 1 type filters actually filters properly
* [x] verify that enabling all filters shows all orders
* [x] verify that no filters shows all items *after disabling all filters*
